encrypted-media/scripts/playback-persistent-license-events.js

```diff
@@ -0,0 +1,168 @@
+function runTest(config, testname) {
+
+    var testname = config.keysystem + ', successful playback, persistent-license, '
+                                    + /video\/([^;]*)/.exec( config.videoType )[ 1 ]
+                                    + ', set src before setMediaKeys, check events';
+
+    var configuration = {   initDataTypes: [ config.initDataType ],
```

Comment:
I don't see any coding style guidelines in the documentation. We can clean this up under #3423.